components/env.rst

```diff
@@ -0,0 +1,110 @@
+.. index::
+   single: Env
+   single: Components; Env
+
+The Env Component
+=================
+
+    The Env Component parses ``.env`` files to make environment variables
+    stored in them accessible via ``getenv()``, ``$_ENV``, or ``$_SERVER``.
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/env`` on `Packagist`_);
```

Comment:
missing `.. _Packagist: https://packagist.org/packages/symfony/env` at the end of this document